ThunderRequest/RequestController.swift

```diff
@@ -230,7 +230,7 @@ public final class RequestController {
     ///   - headers: (Optional) a dictionary of override headers to be merged with `sharedRequestHeaders`
     ///   - completion: (Optional) A closure to be called once the request has completed
     /// - Returns: The request object that will be run
-    @discardableResult public func request(
+    @discardableResult open func request(
```

Comment:
Does these need to be open? I'd rather keep them if possible as the only real use-case I can think of for overriding them is subclassing `RequestController` and unsure why that'd be necessary :)

Reply:
At the moment, it'd be good to have these open following the same pattern as `StormLanguageController` so it can be used within unit tests.

However, 100% agree with you that it isn't ideal - which is why next week I'm going to look into using protocols which these controller-type classes can conform to - that way these classes can stay as final & we can use the protocols to create mocks for testing :D